lib/cogctl/actions/bundles/create.ex

```diff
@@ -63,7 +63,7 @@ defmodule Cogctl.Actions.Bundles.Create do
         end
 
         commands = for command <- bundle.commands do
-          [command["name"], command["id"]]
+          [command.name, command.id]
```

Comment:
Technically it doesn't. The dep for cog-api was out of date. When I made the changes cog-api necessary for this PR, this broke. I could have made another PR just for this, but because it was such a small change, and cogctl wasn't currently broken, I didn't.